docs/content/2.deploy/20.providers/cloudflare.md

```diff
@@ -300,26 +242,20 @@ NITRO_HELLO_THERE="captain"
 SECRET="top-secret"
 ```
 
-## Advanced
+## Cloudflare Bindings
```

Comment:
Here I just included the Bindings and not the rest of the Cloudflare context (so the request and ctx object), I wonder if I should 🤔 (on one hand it would make these docs more comprehensive, on the other I wouldn't want to make these too overwhelming)